src/keycloak-theme/login/common/services/dynamic-style-loader.service.ts

```diff
@@ -1,13 +1,49 @@
-import { inject, Injectable, Renderer2, RendererFactory2 } from "@angular/core";
-import { Observable } from "rxjs";
+import {
+    inject,
+    Injectable,
+    isDevMode,
+    Renderer2,
+    RendererFactory2
+} from "@angular/core";
+import { KcContext } from "keycloakify/login/KcContext";
+import { catchError, forkJoin, Observable, of, switchMap } from "rxjs";
+import { KC_CONTEXT } from "../providers/keycloak-context.provider";
 
 @Injectable({
     providedIn: "root"
 })
 export class DynamicStyleLoader {
     private renderer: Renderer2 = inject(RendererFactory2).createRenderer(null, null);
-    loadStyle(url: string): Observable<void> {
+    kcContext: KcContext = inject(KC_CONTEXT);
+    loadStyle() {
+        let stylesheets = [
+            `${this.kcContext.url.resourcesCommonPath}/node_modules/@patternfly/patternfly/patternfly.min.css`,
+            `${this.kcContext.url.resourcesCommonPath}/node_modules/patternfly/dist/css/patternfly.min.css`,
+            `${this.kcContext.url.resourcesCommonPath}/node_modules/patternfly/dist/css/patternfly-additions.min.css`,
+            `${this.kcContext.url.resourcesCommonPath}/lib/pficon/pficon.css`,
+            `${this.kcContext.url.resourcesPath}/css/login.css`
+        ];
+        if (isDevMode()) {
+            stylesheets = stylesheets.map(s => `/static/media${s}`);
```

Comment:
Get the resources from assets in local mode

Reply:
But this is only relevant for Production mode? Have you tried this?

Reply:
it isn't.
it is defined in build.options.
you can override configurations in configurations.production, configurations.development etc...

Reply:
if you want to try just run npm serve with these changes (I've already do it on local and on docker)

Reply:
no it does not work in locale

Reply:
![image](https://github.com/user-attachments/assets/3c0b6eb2-2bb9-4c16-a04c-6a96f375c103)
![image](https://github.com/user-attachments/assets/3c0b6eb2-2bb9-4c16-a04c-6a96f375c103)

Reply:
it works... have you run `npx keycloakify copy-keycloak-resources-to-public`?

Reply:
if you have errors on compilation or at runtime please past them here